test/FsAutoComplete.Tests.Lsp/ExtensionsTests.fs

```diff
@@ -148,7 +171,10 @@ let linterTests state =
       Source = "F# Linter"
       Message = "`a <> true` might be able to be refactored into `not a`."
       RelatedInformation = None
-      Tags = None }
+      Tags = None
+      Data = None
+      CodeDescription = None
```

Comment:
I expect most of these changes to fail comparison, waiting for test results to adjust baselines.

Reply:
fixed up